costs/costs_route.go

```diff
@@ -191,15 +191,20 @@ func makeElasticSearchRequestAndParseIt(ctx context.Context, parsedParams esQuer
 	)
 	res, err := searchService.Do(ctx)
 	if err != nil {
+		if err.Error() == "elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]" {
```

Comment:
Instead of comparing the error string, you should use `github.com/olivere/elastic.v5.IsNotFound`.